src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturoteca.controllers;
+
+import java.util.*;
+
+import culturoteca.exception.VideoNotFoundException;
+import culturoteca.model.Video;
+import culturoteca.service.impl.CultureMediaServiceImpl;
+
+public class CultureMediaController {
+
+	private final CultureMediaServiceImpl cultureMediaService;
+
+
+	public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
+		this.cultureMediaService = cultureMediaService;
+	}
+
+
+	public List<Video> find_allVideos() throws VideoNotFoundException {
+		List<Video> videos = null;
```

Comment:
Simplify the findAllVideos Method, instead of starting the videos list as null and reassigning it later, it is better to directly return the result of cultureMediaService.findAll().